src/main/resources/application.properties

```diff
@@ -0,0 +1,8 @@
+spring.datasource.url=jdbc:postgresql://localhost:5432/cartservice
```

Comment:
Use environment variables. Don't hardcode sensitive information like passwords and username

Reply:
ok understood